### ISSUE TITLE
How to install os-se from usb drive (dec added)

### ISSUE THREAD

**Issue by Philip2809** [procount/pinn#509]
I am new to linux and I need to install 2 oses to my Rasbperry Pi 3 B. I have two images, one android version and a kodi. I have tried to convert the images to tar.xz files but it still didnt recognize the files. I was not able to find anything about the JSON files you need, the supporting JSON files, where do I find them? Do i have to make them myself? I also would like to know if I made the tar.xz files correctly, I used a Ubunto command line that I got from the Windows Store. I used this command:
`tar cvf $HOME/rtl_archive.tar /usr/src`
How do I solve this problem?

**Comment by lurch** [comment]
I think you should just use the online-OSes supplied when you boot up PINN with an internet connection, rather then try to mess around with "converting" your own images (which is quite a complex process).

**Comment by lurch** [comment]
@procount Is the similarity here to #482 just coincidence? Or are people somehow misunderstanding PINN's documentation?

**Comment by Philip2809** [comment]
Hi, I just relized that the Os-es I needed was already there now that i did the internet connection. I tought it was gonna be the same offering as in NOOBS, I dont know if I missed that port of the docs where it says that you already uploaded a lot of oses, maybe give a list of which ones is already provieded when you add a network connection? I solved my problems but thx for the fast response and explanation!
